proto/Peer.proto

```diff
@@ -2,15 +2,106 @@ syntax = "proto3";
 
 package peer;
 
-message HandshakeMessage {
-  bytes target_pub_key = 1;
-  bytes requesting_pub_key = 2;
-  bytes message = 3;
-  bytes response_peer_info = 4;
+//////////////////
+// Peer Service //
+//////////////////
+service Peer {
```

Comment:
So if we move towards using GRPC entirely. This would mean that the pk client, polykey gui client  and the pk-agent are all using the same GRPC service. So we can still namespace the protobuf schema, but ultimately they are all communicating over the same GRPC system.

Reply:
yeah well actually what we can do is add multiple services, so we just define one `grpc.Server` instance and can have those services defined on the one server.

Reply:
added a comment to this effect: https://github.com/MatrixAI/js-polykey/issues/90#issuecomment-691781129